src/expandableCalendar/weekCalendar.js

```diff
@@ -57,62 +61,133 @@ class WeekCalendar extends Component {
       dayOfTheWeek = 7 + dayOfTheWeek;
     }
     // leave the current date in the visible week as is
-    const dd = index === 0 ? d : d.addDays(firstDay - dayOfTheWeek);
-    const newDate = dd.addWeeks(index);
+    const dd = weekIndex === 0 ? d : d.addDays(firstDay - dayOfTheWeek);
+    const newDate = dd.addWeeks(weekIndex);
     const dateString = newDate.toString('yyyy-MM-dd');
     return dateString;
   }
 
-  onViewableItemsChanged = ({viewableItems}) => {
-    // NOTE: gets called only on week scroll (so might be un-synced with displayed week) !!!
-    if (viewableItems.length > 0) {
-      const viweableItem = _.first(viewableItems);
-      if (viweableItem.isViewable && this.visibleItem !== viweableItem.item) {
-        if (this.visibleItem) { // to avoid invoke on first init
-          _.invoke(this.props.context, 'setDate', viweableItem.item, UPDATE_SOURCES.WEEK_SCROLL); 
+  onScroll = ({
```

Comment:
You prettier is breaking the lines where it shouldn't...

Reply:
anyway, fixed

Reply:
I don't. Never got around to it